extras/smoke-test.sh

```diff
@@ -30,10 +30,10 @@ function wait_for(){
 }
 
 # stop the server on any errors
-trap 'kill -9 $server_pid && exit 1' ERR
+trap 'kill $server_pid && exit 1' ERR
 
 # spin up the OpenAPI server in the background
-python -m vllm.entrypoints.openai.api_server --port $HTTP_PORT &
+python -m vllm.entrypoints.openai.api_server --port $HTTP_PORT --model facebook/opt-125m --enforce-eager &
```

Comment:
Is `--enforce-reager` reasonable for smoke testing?

Reply:
faster to load and a default for most tests,  not sure if it's worth the effort to trace opt-125m each time